### PR TITLE
Added files related to Compose page

### PR DESCRIPTION
**Pull request by jainsid2305**
I have completed the assigned work. This PR closes the issue #5  .

This is the UI of the Composing a Complaint page.

After typing the complaint and adding image, This page look like this.

### CODE REVIEW

**Comment by Kenny-08** [comment]
Good job @jainsid2305 ... one suggestion : I guess white color for title would look better and if that card for filling complaints, if shaped a bit rounded at the top right corner may go well with the design.

**Review by shubhanshu02 — APPROVED** [review]
I agree with @Kenny-08, rest everything looks pretty good. Well done @jainsid2305!

File: lib/DropDown.dart [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    return Container(
+      //width: 0.65*MediaQuery.of(context).size.width,
+      height: 75.0,
+      padding: EdgeInsets.all(10.0),
+        child: Center(
+          child: DropdownButton<CategoryItem>(
+              hint: Container(
+                padding: EdgeInsets.symmetric(horizontal: 11),
+                child: Text(
+                  'Categories',
```
Comment (shubhanshu02) [inline lib/DropDown.dart:75]: I guess this should have been "Category"

**Comment by jainsid2305** [comment]
Thanks .
I will make the suggested changes.

File: pubspec.yaml [hunk 2]
```diff
@@ -23,6 +23,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  image_picker: ^0.6.7+7
```
Comment (shubhanshu02) [inline pubspec.yaml:26]: Duplicate key

Comment (shubhanshu02) [reply pubspec.yaml:26]: Remove the duplicates from the 'pubspec.yaml' file and run `flutter pub get` to check if it works

**Pull request merged**